code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,652 @@
+		"Kestrel",
+		"Beacon",
+		"Signal",
+		"Freedom",
+		"Glory",
+		"Axiom",
+		"Eternal",
+		"Icarus",
+		"Harmony",
+		"Light",
+		"Discovery",
+		"Endeavour",
+		"Explorer",
+		"Swift",
+		"Dragonfly",
+		"Ascendant",
+		"Tenacious",
+		"Pioneer",
+		"Hawk",
+		"Haste",
+		"Radiant",
+		"Luminous",
+		"Gallant",
+		"Dependable",
+		"Indomitable",
+		"Guardian",
+		"Resolution",
+		"Fearless",
+		"Amazon",
+		"Relentless",
+		"Inspire",
+		"Implacable",
+		"Steadfast",
+		"Leviathan",
+		"Dauntless",
+		"Adroit",
+		"Mistral",
+		"Typhoon",
+		"Titan",
+		"Kupua",
+		"Alchemist",
+		"Cuirass",
+		"Citadel",
+		"Rondelle",
+		"Camail",
+		"Ocrea",
+		"Ram",
+		"Crest",
+		"Tanko",
+		"Pommel",
+		"Kissaki",
+		"Cavalier",
+		"Anelace",
+		"Flint",
+		"Xiphos",
+		"Parrot",
+		"Chamber",
+		"Annellet",
+		"Cestus",
+		"Talwar")
+	var/list/destination_names = list()	//Names of static holdings that the organization's ships visit regularly.
+	var/autogenerate_destination_names = TRUE
+
+/datum/lore/organization/New()
+	..()
+	if(autogenerate_destination_names) // Lets pad out the destination names.
+		var/i = rand(6, 10)
+		var/list/star_names = list(
+			"Sol", "Alpha Centauri", "Sirius", "Vega", "Regulus", "Vir", "Algol", "Aldebaran",
+			"Delta Doradus", "Menkar", "Geminga", "Elnath", "Gienah", "Mu Leporis", "Nyx", "Tau Ceti",
+			"Wazn", "Alphard", "Phact", "Altair", "Mauna", "Jargon", "Xarxis", "Hestia", "Dalstis", "Cygni", "Haverick", "Corvus", "Sancere", "Cydoni", "Kaliban", "Midway", "Dansik", "Branwyn")
+		var/list/destination_types = list("dockyard", "station", "vessel", "waystation", "telecommunications satellite", "spaceport", "distress beacon", "anomaly", "colony", "outpost")
+		while(i)
+			destination_names.Add("a [pick(destination_types)] in [pick(star_names)]")
+			i--
+
+//////////////////////////////////////////////////////////////////////////////////
+
+// TSCs
+/datum/lore/organization/tsc/nanotrasen
+	name = "Nanotrasen Incorporated"
+	short_name = "Nanotrasen"
+	acronym = "NT"
+	desc = "The largest shareholder in the galactic plasma markets, Nanotrasen is a research and mining corporation which specializes in\
+	 FTL technologies and weapon systems. Frowned upon by most governments due to their shady business tactics and poor ethics record,\
+	  Nanotrasen is often seen as a necessary evil for maintaining access to the often volatile plasma market. Nanotrasen was originally\
+	   incorporated on Earth with their headquarters situated on Mars, however they have recently moved most of their operations to the Epsilon Eridani sector."
+	history = "" // To be written someday.
+	work = "research giant"
+	headquarters = "Mars"
+	motto = ""
+
+	ship_prefixes = list("NSV" = "exploration", "NTV" = "hauling", "NDV" = "patrol", "NRV" = "emergency response")
+	// Note that the current station being used will be pruned from this list upon being instantiated
+	destination_names = list(
```

Comment:
Intended to be a replacement.

destination_names = list(
		"NAS Trurl in Epsilon Eridani",
		"NAS Crescent in Tau Ceti",
		"NSS Exodus in Tau Ceti",
		"NSS Antiqua in Darsing",
		"NRS Orion in Sol",
		"NSS Vector in Omicron Ceti",
		"NBS Anansi in Omicron Ceti",
		"NSS Redemption in Sirius",
		"NDS Inferno in Tau Ceti",
		"NAB Smythside Central Headquarters on Earth",
		"NAB North Cimmeria Central Offices on Mars",
		)